src/main/scala/cromwell/engine/db/LiquibaseUtils.scala

```diff
@@ -2,14 +2,19 @@ package cromwell.engine.db
 
 import java.sql.Connection
 
-import liquibase.database.DatabaseConnection
 import liquibase.database.jvm.{HsqlConnection, JdbcConnection}
+import liquibase.database.{Database, DatabaseConnection, DatabaseFactory}
+import liquibase.diff.compare.CompareControl
+import liquibase.diff.{DiffGeneratorFactory, DiffResult}
 import liquibase.resource.ClassLoaderResourceAccessor
 import liquibase.{Contexts, LabelExpression, Liquibase}
 import org.hsqldb.persist.HsqlDatabaseProperties
 
 import scala.collection.JavaConverters._
 
+/**
+  * Collection of utilities for liquibase.
+  */
```

Comment:
I personally don't think this comment adds any value, I think `LiquibaseUtils` speaks for itself.